### DISCUSSION TITLE
Jsx won't render twice.

### DISCUSSION THREAD

**Discussion by sksteve128** [solidjs/solid#1134] [General]
When I was doing the tutorial on [jsx](https://www.solidjs.com/tutorial/introduction_jsx), I tried to render the `<div>` object twice, but it doesn't seem to work. Does anyone know any fixes to this? Thanks.

<img width="1301" alt="Screen Shot 2022-07-25 at 4 39 49 PM" src="https://user-images.githubusercontent.com/67932164/180892948-8237ed5a-a1c6-47d6-bd91-c34f5b5d3690.png">

**Comment by ryansolid** [comment]
The DOM only lets you attach the same element in a single location. If you want 2 divs I suggest making svg a function and calling it twice in the JSX.
